process/transaction/interceptor.go

```diff
@@ -81,10 +87,17 @@ func NewTxInterceptor(
 // ProcessReceivedMessage will be the callback func from the p2p.Messenger and will be called each time a new message was received
 // (for the topic this validator was registered to)
 func (txi *TxInterceptor) ProcessReceivedMessage(message p2p.MessageP2P) error {
+	canProcess := txi.throttler.CanProcess()
```

Comment:
todo: check if throttler needs to be applied also on the processtransaction go routine.

Reply:
Added todo near the go routine